src/axiosCromwell.ts

```diff
@@ -1,10 +1,51 @@
 import axios from 'axios';
+import { useDispatch } from "react-redux";
+import { sessionExpired } from "./store/auth/actions";
 
 const axiosCromwell = axios.create({
-  baseURL: 'http://localhost:3001/',
+  baseURL: 'http://localhost:8080/',
   headers: {
     'Content-Type': 'application/json'
   }
 });
 
+axiosCromwell.interceptors.request.use(
+  config => {
+    const accessToken = localStorage.getItem('accessToken');
+    if (accessToken) {
+      config.headers["Authorization"] = accessToken;
+    }
+    return config;
+  },
+  error => {
+    Promise.reject(error);
```

Comment:
Probably, you wanted to return this? You can write this in a short form: `error => Promise.reject(error)`, or with eta-elimination — just `Promise.reject`.

Reply:
Thank you!

I was interested in finding more information about `eta-elimination` (just to be sure that I understood it correctly) but I haven't found any description. Could you please help me with it?